content-test/selectors/selectors.test.js

```diff
@@ -51,18 +51,29 @@ describe("selectors", () => {
   describe("selectSpotlight", () => {
     let state = selectSpotlight(fakeState);
 
+    function setup(custom) {
+      state = selectSpotlight(Object.assign({}, fakeState, custom));
+      return state;
+    }
+
+    beforeEach(() => setup());
+
     // Tests that provided sites don't get selected, because they don't
```

Comment:
Is this comment valid? `selectSpotlight` Does not filter things out it just sorts them in the order they should be selected, and the assert in the function below tests that length is increased by 2 so everything is selected.